### DISCUSSION TITLE
Is it possible to change language of the app?

### DISCUSSION THREAD

**Discussion by joaoribeirovaz** [aonez/Keka#1100] [Q&A] [answered]
Hi,
I would prefer to have the app in a different language from the system setting, is it possible?

(I found a [similar question](https://forum.keka.io/viewtopic.php?t=1560) on the old forum, but I could not find the plist it mentions).

Thanks,
João

**Comment by gingerbeardman** [comment]
Did you try the tip at the bottom of that thread?

> For the future, the correct way is to set your Mac language to German and English as the second language on the list (In the System Preferences/Language & Text list). You don't really need to logout since you only want to change Keka language, so restarting Keka is sufficient.

https://forum.keka.io/viewtopic.php?p=1706#p1706

**Comment by aonez** [comment]
@joaoribeirovaz just use this command in the Terminal.app and restart Keka:

```
defaults write com.aone.keka AppleLanguages "(en)"
```

Change `en` with whatever you want. For example `de` if you want it to launch in German or `ja` if you want it in Japanese.

To use the system setting simply remove that preference:

```
defaults remove com.aone.keka AppleLanguages
```

**Reply by gingerbeardman** [reply]
Nice! This should be in the Wiki.

**Reply by joaoribeirovaz** [reply]
Thanks! This worked (I had to remove the ‘ from your code).
`defaults write com.aone.keka AppleLanguages "(en)"`

**Reply by aonez** [reply]
Ups! Edited and removed the extra `. Also marked the answer below as the correct one since most users will welcome that less obscure option.

**Comment by aonez** [comment, marked as answer]
Even better if you're running macOS Catalina or newer you can do this in the  Preferences:

1.  Preferences
2. Languages & Region
3. Apps

https://user-images.githubusercontent.com/480163/167868360-567cf3eb-7eca-496b-bacd-f5ddb68ff787.mov

I just found out this option even existed.
